src/server/bitops_family_test.cc

```diff
@@ -0,0 +1,76 @@
+#include "server/bitops_family.h"
+
```

Comment:
header

Reply:
OK

Reply:
For some reason I cannot reply for some of the comments, but I general I've changed all the requests (specifically  with the headers and the casting). For some other reason its not showing here that some of it is outdated (maybe the push -f ?).